client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -152,8 +152,8 @@ class JetpackConnectNotices extends Component {
 				return noticeValues;
 			case 'userIsAlreadyConnectedToSite':
 				noticeValues.text = translate(
-					'This WordPress.com account is already connected to an other user on this site. ' +
-						'Please login to an other WordPress.com account to complete the connection.'
+					'This WordPress.com account is already connected to another user on this site. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'This WordPress.com account is already connected to an other user on this site. Please login to an other WordPress.com account to complete the connection.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).